zap/src/main/java/org/zaproxy/zap/extension/api/ContextAPI.java

```diff
@@ -465,6 +466,18 @@ private ApiResponse buildResponseFromContext(Context c) {
         fields.put("inScope", Boolean.toString(c.isInScope()));
         fields.put("excludeRegexs", jsonEncodeList(c.getExcludeFromContextRegexs()));
         fields.put("includeRegexs", jsonEncodeList(c.getIncludeInContextRegexs()));
+        fields.put(
+                "includedTechnologyList",
```

Comment:
`includedTechnology`

Reply:
Would it be ok to change the field name to `includedTechnologies ` as similar to `excludeRegexs`?

Reply:
Sounds good.